docs/guides/vs-code-debugging.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Debugging With VS Code
+sidebar_label: VS Code Debugging
+description: How to debug a Stencil component using VS Code
+slug: /vs-code-debugging
+---
+
+# Debugging With VS Code
+
+VS Code offers a streamlined debugging experience that can be started with a single click when using [launch configurations](https://code.visualstudio.com/docs/editor/debugging#_launch-configurations).
+
+For more information on how to use the VS Code debugger, please visit the [debugger documentation](https://code.visualstudio.com/docs/editor/debugging).
```

Comment:
This sentence can read a little funky - since folks are coming here to figure out how to use the debugger with Stencil, we might want to change this to specify the link is for general usage of a debugger:
```suggestion
If you're unfamiliar with using the VS Code debugger in general, please visit the [VS Code debugger documentation](https://code.visualstudio.com/docs/editor/debugging) for a primer.
```

Reply:
Yup, like how that sounds